### PR TITLE
Add groups methods

### PR DESCRIPTION
**Pull request by mdb**
Addresses issue #13

### CODE REVIEW

File: src/urls.js [hunk 1]
```diff
@@ -51,6 +51,18 @@ class Urls {
...
+  getGroupActivity(id, query) {
+    return `${this.group(id)}/activity${this.queryString(query)}`;
+  }
+
+  getGroupAdmins(id) {
+    return `${this.groupsBase()}/${id}/admins`;
+  }
+
+  getGroupMembers(id) {
+    return `${this.groupsBase()}/${id}/members`;
```
Comment (mitruly) [inline src/urls.js]: `getGroupMembers` should also accept a query string (source: https://github.com/vinyldns/vinyldns/blob/44eed12b08510750a7688714159be9e6d8319ed9/modules/api/src/main/scala/vinyldns/api/route/MembershipRouting.scala#L124-L127).

File: src/urls.js [hunk 2]
```diff
@@ -51,6 +51,18 @@ class Urls {
     return `${this.groupsBase()}/${id}`;
   }
 
+  getGroupActivity(id, query) {
+    return `${this.group(id)}/activity${this.queryString(query)}`;
+  }
+
+  getGroupAdmins(id) {
+    return `${this.groupsBase()}/${id}/admins`;
```
Comment (mitruly) [inline src/urls.js]: `${this.groupsBase()}/${id}` in `getGroupAdmins` and `getGroupMembers` can also be replaced by `${this.group(id)}`

Comment (mdb) [reply src/urls.js]: Ah, good call!

File: src/urls.js (hunk 1 above)
Comment (mdb) [reply src/urls.js]: Thanks! I overlooked that.

**Comment by mdb** [comment]
@mitruly Thanks for the review! I believe I addressed your comments in commit `a7b1d6c`.

**Pull request merged**